.fx/settings.json

```diff
@@ -14,5 +14,7 @@
             "fx-resource-aad-app-for-teams",
             "fx-resource-simple-auth"
         ]
-    }
+    },
+    "currentEnv": "default",
+    "projectId": "ada57d22-2e1d-4467-8ed1-16bcaf38e044"
```

Comment:
Can this be changed back to the original files?

Reply:
Yup this too, in addition to .fx/env.default.json